reference-implementation/to-upstream-wpts/transform-streams/general.js

```diff
@@ -300,4 +300,36 @@ promise_test(() => {
   return readableStreamToArray(ts.readable);
 }, 'methods should not not have .apply() or .call() called');
 
+promise_test(t => {
+  const transformer = {
+    transform() {
+      transformer.transform = undefined;
+      throw new TypeError();
+    }
+  };
+  const ts = new TransformStream(transformer);
+  return Promise.all([
+    promise_rejects(t, new TypeError(), ts.writable.getWriter().write('a'), 'write() should throw'),
+    promise_rejects(t, new TypeError(), ts.readable.getReader().read(), 'read() should throw')
```

Comment:
I confused myself. If we fallback there is no exception thrown at all. So it is testable. I switched to using thrownError instead of TypeError as I now have another test for implementations which try to be clever about checking the exception.